.github/workflows/test-bun.yml

```diff
@@ -0,0 +1,38 @@
+name: Tests
+# only one can run at a time
+concurrency:
+  # If PR, cancel prev commits. head_ref = source branch name on pull_request, null if push
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  push:
+    # We intentionally don't run push on feature branches. See PR for rational.
+    branches: [unstable, stable]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  unit-tests-bun:
+    name: Unit Tests (Bun)
+    runs-on: buildjet-4vcpu-ubuntu-2204
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: yarn
+      - uses: oven-sh/setup-bun@v2
+        with:
+          bun-version: latest
+      - name: Install Yarn
+        run: bun install -g npm:yarn
+      - name: Install
+        run: yarn install --frozen-lockfile
+      - name: Build
+        run: yarn build
+      - name: Unit Tests
+        # These packages are not testable yet in Bun because of `@chainsafe/blst` dependency.
```

Comment:
i thought chainsafe/blst works on bun?

Reply:
It failed with following error for following packages. I would investigated and (fixed) it separately to have clear understanding what was the problem and fix. 

```
panic(main thread): Segmentation fault at address 0x0
Crashed while loading native module: /Users/nazar/Hub/Lodestar/dev/lodestar-2/packages/light-client/node_modules/@chainsafe/blst/prebuild/darwin-arm64-127-binding.node
oh no: Bun has crashed. This indicates a bug in Bun, not your code.
```